src/TwitterStream.d.ts

```diff
@@ -1,5 +1,15 @@
+export interface StreamOptions {
+  // Number of seconds to wait for data or a heartbeat ping from Twitter before
+  // considering the stream closed (default value is 30 seconds).
+  timeout: number | undefined;
```

Comment:
NIT: this parameter is expressed in seconds, but the implementation in `TwitterStream` uses milliseconds. Either update the comment or logic.